backend/src/main/kotlin/ch/learntrack/backend/admin/grade/GradeService.kt

```diff
@@ -0,0 +1,38 @@
+package ch.learntrack.backend.admin.grade
+
+import ch.learntrack.backend.common.EntityService
+import ch.learntrack.backend.common.LearnTrackConflictException
+import ch.learntrack.backend.grade.GradeDto
+import ch.learntrack.backend.persistence.tables.daos.GradeDao
+import ch.learntrack.backend.persistence.tables.pojos.Grade
+import ch.learntrack.backend.persistence.tables.records.GradeRecord
+import java.time.LocalDateTime
+import java.util.UUID
+
+public class GradeService(private val gradeDao: GradeDao) : EntityService<GradeDto, GradeRecord, Grade>(gradeDao) {
+    public override fun mapToDto(pojo: Grade): GradeDto = GradeDto(
+        id = pojo.id,
+        name = pojo.name,
+        schoolId = pojo.schoolId,
+    )
+
+    public fun createGrade(createGradeDto: CreateGradeDto): Grade {
+        gradeDao.isGradeAlreadyExistingInSchool(createGradeDto)?.let {
+            throw LearnTrackConflictException(
+                "Grade with name ${createGradeDto.name} already exists for school with id ${createGradeDto.schoolId}",
```

Comment:
```suggestion
                "Grade ${createGradeDto.name} already exists for school ${createGradeDto.schoolId}",
```
as these comments only we can see in the logs on the server and therefore keep them short and simple while keep important information